### PR TITLE
Update release.yml

### PR DESCRIPTION
**Pull request by Loyalsoldier**
Remove all files except `README.md`, files in `.git` and `.github` directory.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
           DIST_TAGS_URL="https://api.github.com/repos/${{ github.repository }}/tags"
           DIST_TAG_FOUND=`curl -s ${DIST_TAGS_URL} --connect-timeout 5 | grep "\"name\"\: \"${CORE_LATEST_VER}\""` || true
           if [ -z ${DIST_TAG_FOUND} ]; then
-            rm -rf *.zip *.dgst
+            find . -type 'f' | grep -v "README.md" | grep -v ".git" | grep -v ".github" | xargs rm -rf
```
Comment (rikkihamano) [inline .github/workflows/release.yml:32]: A small problem:
`.git` and `.github` are directories, not files, so
```
grep -v ".git" | grep -v ".github"
```
does not make sense.

Comment (Loyalsoldier) [reply .github/workflows/release.yml:32]: Files in `.git` and `.github` will be removed if not. We need to keep files in `.git` and `.github` directory. Not only the directory structure itself.

Comment (Loyalsoldier) [reply .github/workflows/release.yml:32]: I tested it in macOS `find`. It will list files in sub-directories too.

**Comment by Loyalsoldier** [comment]
Don't merge. I will force push another solution to use `find` to exclude paths.

File: .github/workflows/release.yml (hunk 1 above)
Comment (Loyalsoldier) [reply .github/workflows/release.yml:32]: I will use `find` to exclude paths in a new PR.

**Pull request closed without merging**